cowprotocol/cowswap
# Removed google analytics id

# Summary

It's now defined on GH and Vercel

  # To Test

1. This PR should use the test key (configured on Vercel)
2. The prod key should be used when deployed to barn, staging or prod

*Note:* Keys are configured on infrastructure settings, so they will be used on next deployment regardless of this change
